llx/builtin_simple.go

```diff
@@ -94,11 +94,28 @@ func boolOrOpV2(e *blockExecutor, bind *RawData, chunk *Chunk, ref uint64, fLeft
 // boolAndOp behaves like boolOp, but checks if the left argument is false first
 // (and stop if it is). Only then proceeds to check the right argument.
 func boolAndOpV2(e *blockExecutor, bind *RawData, chunk *Chunk, ref uint64, fLeft func(interface{}) bool, fRight func(interface{}) bool) (*RawData, uint64, error) {
+	arg := chunk.Function.Args[0]
+
 	if bind.Value != nil && !fLeft(bind.Value) {
+		// if the left side is false, we can return false immediately
```

Comment:
can we add a test for this? if we put an invalid query/something that panics in the right side to verify it never gets evaluated